### PR TITLE
Test infrastructure + sample test for apps endpoint

### PR DESCRIPTION
**Pull request by brandur**
Add some basic testing infrastructure in `test_helper` similar to what
we have in API and add a working test for the apps endpoint.

### CODE REVIEW

File: .env.test [hunk 1]
```diff
@@ -1 +1 @@
-DATABASE_URL=postgres://localhost/myapp-test
+DATABASE_URL=postgres://localhost/pliny-test
```
Comment (brandur) [inline .env.test:1]: @pedro Made this Pliny for now, but let's change this around when we add the custom naming system for `lib/`.

File: lib/pliny/utils.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+module Pliny
+  module Utils
```
Comment (brandur) [inline lib/pliny/utils.rb:2]: @pedro Don't want to start another Core-style Utils, but I added a couple minimal functions here for now. We should definitely keep an eye on it. Thoughts?

Comment (pedro) [inline lib/pliny/utils.rb:2]: This looks reasonable! we'll see how it evolves

File: .env.test (hunk 1 above)
Comment (pedro) [inline .env.test:1]: cool!

**Comment by brandur** [comment]
Thanks for taking a look!

**Pull request merged**